workloads/kube-burner/common.sh

```diff
@@ -22,6 +22,7 @@ export TOLERATIONS="[{key: role, value: workload, effect: NoSchedule}]"
 export UUID=${UUID:-$(uuidgen)}
 export OPENSHIFT_VERSION=$(oc version -o json | jq -r '.openshiftVersion') 
 export NETWORK_TYPE=$(oc get network.config/cluster -o jsonpath='{.status.networkType}') 
+export INGRESS_DOMAIN=$( oc get IngressController default -n openshift-ingress-operator -o jsonpath='{.spec.domain}')
```

Comment:
Looking at a couple of deployments I have (and others), this doesn't always have a value... 

```
jtaleric at rhino in ~/dev/scratch/kconfig $ oc get IngressController default -n openshift-ingress-operator -o jsonpath='{.spec.domain}'
 at rhino in ~/dev/scratch/kconfig $ oc get IngressController default -n openshift-ingress-operator -o jsonpath="{.spec}" | jq
{
  "clientTLS": {
    "clientCA": {
      "name": ""
    },
    "clientCertificatePolicy": ""
  },
  "httpCompression": {},
  "httpEmptyRequestsPolicy": "Respond",
  "httpErrorCodePages": {
    "name": ""
  },
  "replicas": 1,
  "tuningOptions": {},
  "unsupportedConfigOverrides": null
}

jtaleric at rhino in ~/dev/scratch/kconfig $ oc get clusterversion
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.10.24   True        False         19d     Cluster version is 4.10.24

```

Should we validate this value is set before the script proceeds?

Reply:
That's surprising, I'm not sure why that field wouldn't be populated (it is on my OSD cluster)...Perhaps I should trust good old bash?

oc get routes -A --no-headers | head -n 1 | awk {'print$3'} | cut -d "." -f 2-

Reply:
Fixed